sunkit_pyvista/plotter.py

```diff
@@ -319,6 +323,14 @@ def save(self, filepath, overwrite=False):
         ----------
         filepath : `str` or `pathlib.Path`
             Name of the file to save as, should have vtm or vtmb as an extension.
+
+        Examples
+        --------
+        >>> from sunkit_pyvista import SunpyPlotter
+        >>> plotter = SunpyPlotter()
+        >>> plotter.plot_solar_axis()
+        >>> plotter.save('./filename.vtm') # doctest: +SKIP
```

Comment:
What happens if we dont skip this?

Reply:
Nothing actually, I had a look at how I wrote the save method for mapsequence and skipped it based on the review over at sunpy.